processor/isolationforestprocessor/processor_test.go

```diff
@@ -96,6 +96,12 @@ func Test_newIsolationForestProcessor_Basic(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, p)
 
+	// Add shutdown cleanup
+	t.Cleanup(func() {
+		shutdownErr := p.Shutdown(t.Context())
```

Comment:
```suggestion
		shutdownErr := p.Shutdown(context.WithoutCancel(t.Context()))
```

The t.Context method returns a context that is cancelled when the test ends, before cleanups are invoked. This has been the cause of flaky tests in other components recently.

I see that the Shutdown method ignores context (it shouldn't), but even so it would be good to make this change to avoid others copy & pasting the same thing and causing flaky tests in other components.